pkg/advisory/osv.go

```diff
@@ -148,8 +148,9 @@ func BuildOSVDataset(_ context.Context, opts OSVOptions) error {
 					continue
 				}
 
-				// Note: The OSV data should include our advisory ID itself among the listed aliases.
-				aliases := append([]string{adv.ID}, adv.Aliases...)
+				// Note: The OSV data should include our advisory ID itself among the listed
+				// related vulnerability IDs.
+				related := append([]string{adv.ID}, adv.Aliases...)
```

Comment:
Hi,
I just wanted to ask why you include the advisory's own ID in its `related` (or `aliases`) field.
It seems a bit odd to make a record link to itself, but it seem like an intentional decision?

Reply:
was my understanding in the beginning to add that as is. maybe i got that wrong

Reply:
Yeah it sounded like this was feedback from the OSV team — originally we didn't add in the ID itself to this collection. Happy to remove it if it's not needed! 🧹